Source/NSTextCheckingController.m

```diff
@@ -0,0 +1,130 @@
+/* Implementation of class NSTextCheckingController
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: 02-08-2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import "AppKit/NSTextCheckingController.h"
+
+@implementation NSTextCheckingController
+
+// initializer
+- (instancetype) initWithClient: (id<NSTextCheckingClient>)client
+{
+  self = [super init];
+  if (self != nil)
+    {
+      _client = client;
+      _spellCheckerDocumentTag = 0;
+      _menu = nil;
```

Comment:
What is this instance variable for? It never gets used anywhere.

Reply:
Based on some of the methods I thought it might be needed... if it's not, I will remove it.  I'm assuming you're referring to "_menu"

Reply:
removed

Reply:
No you didn't.

Reply:
I'm not sure why you're not seeing this...
// initializer
- (instancetype) initWithClient: (id<NSTextCheckingClient>)client
{
  self = [super init];
  if (self != nil)
    {
      _client = client;
      _spellCheckerDocumentTag = 0;
    }
  return self;
}

And I have pushed.

Reply:
This should be gone.  Please pull and check on your end.  Re-open this if you still don't see the change.